content/articles/Website-vs-Web-app/index.md

```diff
@@ -0,0 +1,147 @@
+    alt: difference between a Web Page and a Web Applicationexample image
+---
+
+### Introduction
+It is difficult for most people to distinguish between a website and a web application. They enter the URL and get the expected outcomes, making them unaware of the backend processes.
+These products are created for the consumer by developers. Therefore, a developer must be able to tell the difference between the two. 
+Some developers argue that the terms `website` and `web application` are similar, while others say they are not. Who is wrong, and who is correct?😕
+I was once among those who argued that the two terms mean the same, but after research, I found out that these are two different things. 
+In this article, I will highlight some of the main differences that have helped me understand the differences between websites and web applications.
+
+In this tutorial we will look at:
+1. [Website](#website)
+2. [Web application](#web-application)
+3. [Distinctions between a website and a web application](#distinctions-between-a-website-and-a-web-application)
+
+### Website
+
+A website is a collection of interconnected pages that may contain images, audio, or video and are linked by a single domain name.
+
+Website is of different types. Some of them are:
+
+1. Dating websites
+
+2. Educational Websites
+
+3. Social Media websites
+
+4. Portfolio website
+
+#### Characteristics of A website
+
+A good web designer can Judge a site according to its appearance. One of the characteristics that will determine the success of a website is its appearance. Let's have a look at some additional website attributes.
+ 1. ``User friendly``- A website can be operated by almost everybody for it is easy to use and navigate.
+
+2. ``Attractive and easy for all targeted users`` - This fixture helps in attracting all users and gives them a reason for not choosing another website to visit.
+
+3. ``Purpose``- What is the objective of the website's appearance? A good website should provide a purpose and that purpose should be clear to the user within a short period. A user should not take much of his/her time trying to understand the reason for the appearance of a site.
+
+   The unclear purpose may lead to a website not giving back good results.
+ 
+4. ``High speed of loading``- Web designers should consider this fixture as not all end-users are patient. In this world of time, there is no doubt that time is a very important resource and nobody would like his/her time to go to waste. Therefore if a website loads slowly, end-users likely to look for a different website with a high speed.
+
+    There numerous elements that factor into the loading speed of a website, yet everything starts with the hosting quality. Always choose a host that loads first and keep end-users happy.
+
+5. ``Secure``- A good website provides security for both end-users and designers 
+
+#### What is the importance of a website
```

Comment:
I think most of these important can be paraphrased and explained in a better way. Do some research on this and try explaining them in shorter paragraphs. I have illustrated this in the professionalism point.